json-path/src/main/java/com/jayway/jsonpath/internal/JsonContext.java

```diff
@@ -146,8 +146,8 @@ public String jsonString() {
         if(jsonPath != null){
         	return read(jsonPath);
         } else {
-        	jsonPath = compile(path, filters);
-        	cache.put(cacheKey, jsonPath);
+            jsonPath = compile(path, filters);
+            cache.put(cacheKey, jsonPath);
```

Comment:
I don't know how much @kallestenflo cares about formatting, but I suggest you restore it here.